tools/sgen/output/MSFNotificationTokens.generated.swift

```diff
@@ -150,7 +150,7 @@ extension FluentUIStyle {
 
 		// MARK: - outlineColor 
 		open var outlineColor: UIColor {
-			return UIColor(named: "FluentColors/clear", in: FluentUIFramework.resourceBundle, compatibleWith: nil)!
+			return UIColor(named: "FluentColors/clear", in: FluentUIFramework.colorsBundle, compatibleWith: nil)!
```

Comment:
This was an unintentional change - all I did was run sgen after making my isolated token changes, but I didn't make any changes to MSFNotificationTokens in the yaml. I figured it was better for everything to be aligned (as in, maybe someone forgot to check in MSFNotificationTokens.generated.swift in the sgen output and this is fixing that mistake) than for this to continue to be an issue. I can revert if preferred.

Is there another reason this change would have happened?

Reply:
This is a side-effect of two old conflicting PRs intersecting. The first was mine:

The first was @jeaniehuynh's:
- #715
This created tokens for NotificationView using the classic single-bundle approach for colors and images. While this was out in PR, I submitted a separate change:

- #748
This PR separated our resource bundle into two separate bundles, `resourceBundle` for iOS-specific resources, and `colorsBundle` for our cross-platform colors (it was necessary for Swift Package Manager stuff).

The latter never converted the tokens for the former, so it's just been waiting for someone to re-run sgen and get the new values.

The code is already correct in `vnext-prototype`, so there's no harm in this fix. Just a side-effect of the fact that this is the first time anyone's re-run sgen since those two PRs both made it to `vnext-prototype`.